docs/usage.md

```diff
@@ -132,19 +132,19 @@ If `-profile` is not specified, the pipeline will run locally and expect all sof
 
 * `docker`
   * A generic configuration profile to be used with [Docker](https://docker.com/)
-  * Pulls software from Docker Hub: [`nfcore/viralrecon`](https://hub.docker.com/r/nfcore/viralrecon/)
+  * When using Biocontainers pulls Docker containers from quay.io e.g [FastQC](https://quay.io/repository/biocontainers/fastqc)
```

Comment:
Couldn't we just include this line one (under a `biocontainers` section maybe) so that it is not repeated so many times, something such as:
When using Biocontainers all container engines pull containers from quay.io e.g [FastQC](https://quay.io/repository/biocontainers/fastqc)" except Singularity that uses directly downloads Singularity images via https hosted by the [Galaxy project](https://depot.galaxyproject.org/singularity/)

Reply:
Good point 👍🏽 Re-worded and mostly fixed in https://github.com/nf-core/viralrecon/pull/173/commits/dbb36937096edf588747e2ac715169afaf5b2e78